integration/build/build_test.go

```diff
@@ -304,6 +306,8 @@ COPY bar /`
 // docker/for-linux#135
 // #35641
 func TestBuildMultiStageLayerLeak(t *testing.T) {
+	skip.IfCondition(t, versions.LessThan(testEnv.DaemonAPIVersion(), "1.38"),
+		"Don't run on API lower than 1.38 as it has been fixed starting from that version")
```

Comment:
More factually; it's not the API version that fixed it, but the Docker version (it has been back ported to Docker 18.03, so API 1.37 should be good as well https://github.com/docker/docker-ce/pull/449)